Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/DefaultSprayToolOptionsView.kt

```diff
@@ -64,15 +64,15 @@ class DefaultSprayToolOptionsView(rootView: ViewGroup) : SprayToolOptionsView {
                 val sizeTextInt = try {
                     sizeText.toInt()
                 } catch (exp: NumberFormatException) {
-                    Log.d(DefaultSprayToolOptionsView.TAG, exp.localizedMessage!!)
+                    Log.d(TAG, exp.localizedMessage!!)
```

Comment:
I think there are still quite a few !! in the code base which should all get removed (except the ones in testcases). This would also be a nice addition for the detekt, if possible.

Reply:
added :)